src/Avalonia.Base/AvaloniaObject.cs

```diff
@@ -304,6 +305,14 @@ public bool IsSet(AvaloniaProperty property)
 
             VerifyAccess();
 
+            var description = GetDescription(source);
+
+            if (priority == BindingPriority.LocalValue)
```

Comment:
I only just noticed this; why only pass `LocalValue` bindings though the scheduler?